co-authors-plus.php

```diff
@@ -1278,8 +1278,10 @@ function load_edit() {
 	 * @since 3.0
 	 */
 	function filter_views( $views ) {
+		$post_type = get_current_screen()->post_type;
+		$numposts = count_user_posts( wp_get_current_user()->ID, $post_type);
```

Comment:
We'll have to update `$post_type` to `$mine_args['post_type']`.  

I'm also seeing the count off, as it does not include the user's posts before a guest author account is linked.  Only the count of the posts assigned after a guest author is linked to the user is accounted for.

Reply:
The count is being pulled from `$numposts` and printed in line 1303, so it appears to be related.